src/pages/Touringcar/components/MapLayers/__snapshots__/MapLayers.test.tsx.snap

```diff
@@ -365,20 +365,69 @@ FAQ en contactgegevens"
                           class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
                           style="margin-block-end: .5em;"
                         >
-                          Weergave van de aanbevolen en verplichte routes voor touringcars.
+                          Parkeerplaatsen, in- en uitstaphaltes en de toegestane touringcarroute in Amsterdam.
                         </p>
                         <p
                           class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: .5em;"
                         >
                           Let op de doorrijhoogtes.
                         </p>
                       </div>
+                      <div
+                        class="sc-eIcdZJ bzzgoc"
+                      >
+                        <p
+                          class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: 0;"
+                        >
+                          <strong>
+                            LET OP:
+                          </strong>
+                        </p>
+                      </div>
+                      <div
+                        class="sc-eIcdZJ bzzgoc"
+                      >
+                        <p
+                          class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: .5em;"
+                        >
+                          <strong>
+                            Toegestane wegen
+                          </strong>
+                        </p>
+                        <p
+                          class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: .5em;"
+                        >
+                          Vanaf januari 2024 mogen touringcars zwaarder dan 7,5 ton niet meer binnen de S100 rijden. Uitzondering is de S100 zelf en de corridor Weesperstraat – Valkenburgerstraat – Kattenburgerstraat.
+                        </p>
+                      </div>
+                      <div
+                        class="sc-eIcdZJ bzzgoc"
+                      >
+                        <p
+                          class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: .5em;"
+                        >
+                          <strong>
+                            Alleen met ontheffing
+                          </strong>
+                        </p>
+                        <p
+                          class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: .5em;"
+                        >
+                          Deze wegen zijn alléén toegankelijk voor touringcars &gt;7,5 ton mét ontheffing. Een ontheffing is alleen aan te vragen voor vervoer van specifieke doelgroepen (basisschoolleerlingen, mensen met een beperking, ouderen, artiesten) in combinatie met een bezoek aan een educatieve en/of culturele instelling.
```

Comment:
Are the `>` characters from the source code displayed correctly with the conversion here to `&gt;`? #nitpick: the old texts seem to have had white space between the `>` and the `7.5`, whereas this does not. Perhaps consider double checking this to verify that this is consistent--provided that we care.

Reply:
I can confirm that the `>` is correctly displayed in the app.

As far as textual consistency is concerned: that's a discussion to be had with the stakeholders who provide the content.